CMakeLists.txt

```diff
@@ -52,32 +62,29 @@ add_subdirectory(mbed-os)
 ### End of block
 ######################################################################################################
 
-### Set up your project name (necessary everytime)
-project(MbedCEHelloWorld)
-
 ### add executable (necessary everytime)
-add_executable(${CMAKE_PROJECT_NAME} main.cpp)
+# This will compile an elf file named the same as the project with the given source files.
+add_executable(${PROJECT_NAME} main.cpp)
 
 ######################################################################################################
 ### Link libraries block
 #[[For more about this configuration visit wiki page MbedOS-configuration
 https://github.com/mbed-ce/mbed-os/wiki/MbedOS-configuration#configuration-via-cmake-files]]
 
 #[[link MbedOS and its libraries (necessary everytime)]]
-target_link_libraries(${CMAKE_PROJECT_NAME}  mbed-os) 
+target_link_libraries(${PROJECT_NAME} mbed-os)
 
 ### link user library (if needed)
-#target_link_libraries(${CMAKE_PROJECT_NAME} YourLibrary) 
+#add_subdirectory(YourLibrary)
+#target_link_libraries(${PROJECT_NAME} YourLibrary)
+
+# This generates build rules to generate bin/hex files, and to flash the code if you have configured an
+# upload method.
+mbed_set_post_build(${PROJECT_NAME})
```

Comment:
Notes from lines 88 and 89 should be directly after line with default `mbed_set_post_build(${PROJECT_NAME})`

Reply:
Good catch, fixed